src/api/app/controllers/services/webhooks_controller.rb

```diff
@@ -0,0 +1,46 @@
+class Services::WebhooksController < ApplicationController
+  #
+  # This controller is checking permission always only on the base of the github signature
```

Comment:
What are you trying to express here? You use `.can_modify?` below for instance...

Reply:
To document why we skip extract_user and require_login filters! I can remove if you don't like it.